string id
string type
string time
string cmd
r_pose_optional[] waypoints
